packages/react-native-editor/__device-tests__/CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ Using one or a combination of these tools will make it much easier to identify w
 
 -   You'll write any functions needed to interact with the page in the `EditorPage` page object and then call those interactions within the test. The code you'll need to write to actually do the finding will use a combination of
 
--   Appium's spec https://appium.io/docs/en/about-appium/intro/ which you can find examples of a variety of functions under the commands tab
```

Comment:
We don't use Appium 2 yet ([reference](https://github.com/WordPress/gutenberg/blob/bdf728b7bf7105ad7ee576200aa46dc82ff424fb/package.json#LL166C4-L166C8)) so not sure if we should point to this documentation. I haven't found the docs for version 1.x, which makes me think that might have been removed, as version 1.x is no longer maintained 🤔. If so, we could add a note about this, WDYT?

Reply:
That's a good point -- Appium's docs have moved around quite a bit recently with the 2.0 release. I couldn't find the 1.x docs initially, but eventually found them on [GitHub](https://github.com/appium/appium/tree/1.x/docs/en), and updated the link references. 👍